discord/enums.py

```diff
@@ -74,7 +74,7 @@ class VerificationLevel(Enum):
     def __str__(self):
         return self.name
 
-class DefaultMessageNotifications(Enum):
+class MsgNotifications(Enum):
```

Comment:
Still not really sold on this name either (We don't use `Msg` shorthand anywhere). I think `MessageNotifications` might be a better intermediate but again, not really sold on it. Other alternatives might be `GuildMessageNotifications` etc.

I'm not convinced there's a good name for this.

Reply:
How about NotificationLevel?